File: April 1, 2026.js
function fixPrankNumber(arr) {

  const firstDifference = arr[1] - arr[0];
  const secondDifference = arr[2] - arr[1];
  const thirdDifference = arr[3] - arr[2];
  const difference = [firstDifference, secondDifference, thirdDifference].sort((a, b) => a - b)[1];

  for (let i = 1; i < arr.length; i++) {

    if (arr[i] - arr[i-1] != difference) {

      if (arr[i + 1] && (arr[i + 1] - arr[i] == difference)) {
        arr[i - 1] = arr[i] - difference;
      } else {
        arr[i] = arr[i - 1] + difference;
      }
      break;
    }

  }

  return arr;
}

File: April 1, 2026.py
def fix_prank_number(arr):

    first_difference = arr[1] - arr[0]
    second_difference = arr[2] - arr[1]
    third_difference = arr[3] - arr[2]

    difference = sorted([first_difference, second_difference, third_difference])[1]

    for i in range(1, len(arr)):
        if arr[i] - arr[i - 1] != difference:
            if (i + 1 <len(arr)) and (arr[i + 1] - arr[i] == difference):
                arr[i - 1] = arr[i] - difference
            else:
                arr[i] = arr[i - 1] + difference
            break

    return arr